clear-webs.lua

```diff
@@ -0,0 +1,32 @@
+-- Removes webs and frees webbed units.
+-- author: Atomic Chicken
+
+--[====[
+
+clear-webs
+==========
+This script removes all webs that are currently on the map,
+and also frees any creatures who have been caught in one.
+
+Note that it does not affect sprayed webs until
+they settle on the ground.
+
+Usable in both fortress and adventurer mode.
+
+]====]
+
+local webCount = 0
+for i = #df.global.world.items.other.ANY_WEBS-1, 0, -1 do
+  dfhack.items.remove(df.global.world.items.other.ANY_WEBS[i])
+  count = count + 1
```

Comment:
are all instances of `count` supposed to be `webCount`?

Reply:
Clearly a typo inserted while I was wondering whether or not to add a unitCount. No idea how it slipped my notice.